portfolio/src/main/webapp/script.js

```diff
@@ -67,15 +66,48 @@ function showSlide() {
 }
 
 /**
- * Fetches a quote from the Office from the server and adds it to the DOM.
+ * Fetches all the comments from the server and add them to comment-list.
  */
-function getOfficeQuote() {
-  fetch('/data').then(response => response.json()).then((data) => {
+function getComments() {
+
+  const DEFAULT_COMMENT_COUNT = '5';
+  let commentCount = document.getElementById('comment-count').value;
+
+  // If comment count is an empty string, set to default.
+  if (commentCount === "") {
+    commentCount = DEFAULT_COMMENT_COUNT;
+  }
+
+  const url = '/data?count=' + commentCount;
+  console.log(url)
+  fetch(url).then(response => response.json()).then((comments) => {
     
-    // data is json of QuotePerson, so reference its attributes
-    const quoteContainer = document.getElementById('quote-container');
-    quoteContainer.innerText = "\"" + data.quote + "\" - " + data.person;
+    // Empty the list that will recieve the comments.
```

Comment:
receive

Reply:
I don't know why, but I always misspell receive. I can confidently tell you how it is spelled, but I will constantly type it wrong.